module/icp/io/skein_mod.c

```diff
@@ -231,6 +231,19 @@ skein_mod_init(void)
 
 int
 skein_mod_fini(void) {
+	int ret;
```

Comment:
[style] by convention the existing code prefers to name this kind of variable `error` rather than `ret`.